src/Microsoft.Data.SqlClient/src/Interop/Windows/Kernel32/Kernel32Safe.netfx.cs

```diff
@@ -1,41 +1,43 @@
-// Licensed to the .NET Foundation under one or more agreements.
+﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#if NETFRAMEWORK
+
 using System;
-using System.Runtime.ConstrainedExecution;
 using System.Runtime.InteropServices;
 using System.Runtime.Versioning;
 using System.Security;
 using System.Text;
-using System.Threading;
 
-namespace Microsoft.Data.Common
+namespace Interop.Windows.Kernel32
 {
-
-    [SuppressUnmanagedCodeSecurityAttribute()]
-    internal static class SafeNativeMethods
+    /// <remarks>
+    /// Be insanely careful when using methods from this class. They are configured to skip
+    /// runtime security checks.
+    /// </remarks>
+    [SuppressUnmanagedCodeSecurity]
```

Comment:
Are there few enough spots where these methods are called that we could add the suppression at each call site and remove it here without it being a huge burden? Then you would get a compiler error if you add a new unsuppressed call.

Reply:
I don't think we could move it to the call site, since the attribute only applies to classes, methods, delegates. Unless you mean to add it to the method signature for a method that calls the "safe" methods... then I'm not really sure what would happen. Like, I don't know if it applies to all things down the stack or if it only applies to immediate transitions from managed to unmanaged. I think this is likely the easiest method.